scripts/rabbitmq-service.bat

```diff
@@ -126,6 +126,11 @@ set RABBITMQ_EBIN_ROOT=!RABBITMQ_HOME!\ebin
 
 set RABBITMQ_CONFIG_FILE="!RABBITMQ_CONFIG_FILE!"

```

Comment:
I am not sure this `set` is necessary. 
In [`rabbitmq-server.bat`](https://github.com/rabbitmq/rabbitmq-server/blob/rabbitmq-server-691/scripts/rabbitmq-server.bat#L43) is not present.